src/scales/scale.time.js

```diff
@@ -554,7 +554,9 @@ module.exports = function() {
 							datasets[i][j] = timestamp;
 						}
 					} else {
-						timestamps.push.apply(timestamps, labels);
+						for (k = 0; k < labels.length; ++k) {
```

Comment:
you could use `j` here rather than having to declare a new variable to iterate with

Reply:
Thanks for the comment. Now, k is replaced with j.